scripts/acceptance-tests-ide.sh

```diff
@@ -2,46 +2,84 @@
 
 set -euo pipefail
 
-readonly TOOLSMITH_ENV_INPUT="${TOOLSMITH_ENV_INPUT:?must be set}"
-readonly DEFAULT_ORG="${DEFAULT_ORG:?must be set}"
-readonly DEFAULT_SPACE="${DEFAULT_SPACE:?must be set}"
-readonly SKIP_SSL_VALIDATION="${SKIP_SSL_VALIDATION:?must be set}"
-
-declare API_HOST
-readonly API_PORT=443
-readonly USERNAME="admin"
-declare PASSWORD
-readonly CLIENT_ID="admin"
+readonly METADATA_FILE="${METADATA_FILE:?must be set}"
+
+declare CF_API_HOST
+declare CF_USERNAME="admin"
+declare CF_PASSWORD
+
+declare CLIENT_ID="admin"
 declare CLIENT_SECRET
 
+declare DEFAULT_ORG="test"
+declare DEFAULT_SPACE="development"
+
+readonly DEPLOYMENT_DIRECTORY="$(mktemp -d)"
+
 discover_environment() {
   local env_name
-  env_name=$(cat "$TOOLSMITH_ENV_INPUT/name")
+  env_name=$(jq -r .name <"${METADATA_FILE}")
 
-  eval "$(bbl print-env --metadata-file "$TOOLSMITH_ENV_INPUT/metadata")"
+  eval "$(bbl print-env --metadata-file "${METADATA_FILE}")"
 
-  API_HOST="$(jq -r .cf.api_url <"$TOOLSMITH_ENV_INPUT/metadata")"
-  PASSWORD="$(credhub get -n "/bosh-${env_name}/cf/cf_admin_password" -q)"
+  CF_API_HOST="$(jq -r .cf.api_url <"${METADATA_FILE}")"
+  CF_PASSWORD="$(credhub get -n "/bosh-${env_name}/cf/cf_admin_password" -q)"
   CLIENT_SECRET="$(credhub get -n "/bosh-${env_name}/cf/uaa_admin_client_secret" -q)"
 }
 
+prepare_cf_deployment() {
```

Comment:
Thanks! Not sure how I feel about this since it can take a very long time to run. The original intent of the script was to just dump out the vars to paste into IJ. I wonder if this should be broken out into a separate `prepare-cf-environment.sh` or something like that?

Reply:
I considered a few things:
- Removing it, but it didn't scale for our ATs
- Creating a file with the replace ops.yml but we need to get the deployment and every time you want to run the ATs, you have to scale the environment up anyway. The second time it just moves forward so it doesn't take that long to leave it as part of the same script.
- Preparing a different pool with everything scaled up already, but I will have to do more research on that.

I decided to leave it as it was since it has been helpful for me running the tests locally.